src/cascadia/TerminalSettingsEditor/Profiles.h

```diff
@@ -134,6 +134,11 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     struct ProfilePageNavigationState : ProfilePageNavigationStateT<ProfilePageNavigationState>
     {
     public:
+        ProfilePageNavigationState() :
+            _Profile{ nullptr },
+            _Schemes{ nullptr },
+            _WindowRoot{ nullptr } {}
+
```

Comment:
In my testing I found this change to be unneeded.

Reply:
Indeed, good catch! Removed now.